src/annotator/anchoring/pdf.js

```diff
@@ -75,10 +75,20 @@ async function getPageView(pageIndex) {
     // a "pdfPage" property.
     pageView = await new Promise(resolve => {
       const onPagesLoaded = () => {
-        document.removeEventListener('pagesloaded', onPagesLoaded);
+        if (pdfViewer.eventBus) {
```

Comment:
Perhaps drop some of those snazzy comments about versions in here for later reference?